### PR TITLE
feat: Pull latest SDK information from the registry

### PR DESCRIPTION
**Pull request by kamilogorek**
Instead of getsentry/getsentry#2304

### CODE REVIEW

**Review by kamilogorek — COMMENTED** [review]
Looks good. Need a double-check from @mitsuhiko though.

**Review by mitsuhiko — APPROVED** [review]

**Comment by HazAT** [comment]
@tonyo Can you please double check with Team Ops if the location where we store the file is good?
Maybe we want to have something that is generally applicable, right now, the json file is stored in the same folder as the python file.

**Review by tonyo — CHANGES_REQUESTED** [review]

**Comment by kamilogorek** [comment]
@tonyo ignored + forced build

**Review by tonyo — APPROVED** [review]
No idea yet why that travis job fails.

**Comment by tonyo** [comment]
I'll take a look at the travis issue now.

File: src/sentry/utils/distutils/commands/build_js_sdk_registry.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    dump_registry('_registry', data)
+
+
+from .base import BaseBuildCommand
+
+
+class BuildJsSdkRegistryCommand(BaseBuildCommand):
+    description = 'build js sdk registry'
+
+    def run(self):
```
Comment (tonyo) [inline src/sentry/utils/distutils/commands/build_js_sdk_registry.py]: I don't see any harm in overriding the entire `run()` method here.

**Comment by tonyo** [comment]
@kamilogorek @HazAT lgtm now

**Comment by HazAT** [comment]
@tonyo Thanks for your help, I will merge this now to have the latest version of the browser SDK available in the loader.

**Pull request merged**